bodhi/server/models.py

```diff
@@ -2256,8 +2256,9 @@ def new(cls, request, data):
                                f"release value of {up.mandatory_days_in_testing} days"
             })
 
-        log.debug("Setting request for new update.")
-        up.set_request(db, req, request.user.name)
+        if not data.get("from_tag"):
+            log.debug("Setting request for new update.")
```

Comment:
While we're in this place of the code, maybe we could reword a bit this statement but not a must do either